.gitignore

```diff
@@ -1,3 +1,7 @@
 *.pyc
 __pycache__
-venv
\ No newline at end of file
+lmnEnv
```

Comment:
There are a couple conflicts to resolve. One of them is with this file. If you don't know where to do that there is a button on the conversations page.